notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -3624,6 +3624,51 @@ coping with a disability [Neo was somehow blinded and had to compensate by using
 free will [A point was made about free will when Neo answered with "Because I choose to." to Agent Smith's question about why he keeps on fighting against all odds.]
 
 
+movie: Timeline (2003)
+======================
+
+:: Title
+Timeline
+
+:: Description
+Timeline is a 2003 American science fiction adventure based on Michael
+Crichton's 1999 novel of the same name.
+
+Synopsis: A team of present-day archaeology and history students are sent back
+in time to medieval France, to rescue their professor from the middle of a
+battle.
+
+:: Date
+2003-11-26
+
+:: References
+https://en.wikipedia.org/wiki/Timeline_(2003_film)
+
+:: Ratings
+3 <paul>
+
+:: Choice Themes
+time travel [A main novelty of the film concerns Chris and company traveling back in time to 1357 France to rescue his archaeologist father who'd become trapped there, after traveling back in time himself.]
+
+:: Major Themes
+father and son [Chris traveled back in time to save his father, Edward, who'd become trapped in 14th century France.]
+what it is like in medieval times [The viewer is shown a knight in shining armor heavy depiction of what life might have been like in 14th century France.]
+archaeology [The excavation of a medieval French village, named Castlegard, was integral to the plot.]
+the archaeology profession [Professor Edward Johnston was leading an archaeological study of the medieval village of Castlegard.]
+romantic love [Chris was infatuated with Kate. Marek became infatuated with Lady Claire and ultimately stayed behind in the year 1357 to spend his life with her.]
+people meddling in history [Much was made of Chris and his companions going back in time and changing history by saving the woman whose martyrdom had supposedly sparked the Hundred Years War between England and France. Much was likewise made of Professor Johnson supplying the English with Greek fire in their war against the French.]
+war [A medieval era war played out between English and French forces.]
+history [The film follows a fanciful depiction of the start of the very real Hundred Years War between England and France.]
+
+:: Minor Themes
+romanticism of the past [The archaeologist Marek waxed about the past, asserting that men had honor and people cared about each in former ages.]
+traversable wormhole [The time machine exploited a stable wormhole.]
+technology [Professor Johnson taught the medieval English how to use Greek fire in warfare.]
+patriotism [A French knight uttered "This is for France" after dispatching an English foe in battle.]
+the desire for vengeance [Decker wanted revenge against the ITC corporation for their leadership having abandoned him in medieval France.]
+the art of war [French forces laid siege to an English held castle.]
```

Comment:
I guess that could be better motivated. Unless you mean that any siege mentioned in a story is cause to use this theme.

Reply:
Deleted. Will save for future themers to suss out if there was some discussion of siege tactics.